frontend/src/components/Auth/MultiAuthSection.jsx

```diff
@@ -111,7 +111,7 @@ const MultiAuthSection = () => {
                         You can access your account by signing in with one of the options below.
                     </p>
                 </div>
-                {window.location.href.includes("localhost")  && ( // login.gov is only configured to work locally at the moment
+                {window.location.href.includes("localhost") && ( // login.gov is only configured to work locally at the moment
```

Comment:
wonder if we could Vite's `import.meta.env.BASE_URL` 

something like this: 

```jsx
{import.meta.env.BASE_URL.includes('localhost') && (
  // ...rest of code
)}
```

Reply:
@fpigeonjr I had looked into that previously but that env variable requires that it be set manually elsewhere and controlled for each environment which was more overhead than I was looking to add at the time based on the need--under the guise of Minimize & Simplify

Reply:
From experimentation with it previously, the `BASE_URL` is looking for a path starting from `/` (without the hostname) so doesn't work here.  `import.meta.env.MODE === "development" &&` works though so I will replace it with this.

Reply:
🤔 @johndeange I thought the idea was to only show this in local development not our development environment in Azure?

Reply:
Yes my change accomplishes this: `MODE === "development"` is the default mode for running out of the vite development server and so the button will only be shown in this environment.

Some more background: `MODE === "production"` is when we are running in production. `import.meta.env.DEV` is the opposite of `import.meta.env.PROD` but not equal to `MODE === "development"`